src/settings-ui/Settings.UI/Views/FancyZonesPage.xaml

```diff
@@ -35,7 +35,7 @@
                     IsEnabled="{x:Bind Mode=OneWay, Path=ViewModel.IsEnabled}">
                     <labs:SettingsCard
                         x:Uid="FancyZones_LaunchEditorButtonControl"
-                        ActionIcon="{ui:FontIcon FontFamily={StaticResource SymbolThemeFontFamily}, FontSize=14, Glyph=&#xE8A7;}"
```

Comment:
Old vs. new.
![image](https://user-images.githubusercontent.com/9866362/230792296-557e18ec-1b30-4bd3-9641-2ec2ed09cd4a.png)

Revert - this `FontIcon` is rendered within a `Viewbox` with a `MaxWidth`. Increasing its size actually makes it smaller.

Reply:
Alright this one is for you then :)